shoop/core/models/_products.py

```diff
@@ -163,6 +164,7 @@ class Product(TaxableItem, AttributableMixin, TranslatableModel):
     created_on = models.DateTimeField(auto_now_add=True, editable=False, verbose_name=_('created on'))
     modified_on = models.DateTimeField(auto_now=True, editable=False, verbose_name=_('modified on'))
     deleted = models.BooleanField(default=False, editable=False, db_index=True, verbose_name=_('deleted'))
+    archived = models.BooleanField(default=False, editable=False, db_index=True, verbose_name=_('archived'))
```

Comment:
There shouldn't be need to create new attribute for archiving. I think you should use delete for archiving. Also use the soft delete functionality to archive/hide products.